spec/03-plugins/31-proxy-cache/02-access_spec.lua

```diff
@@ -1346,5 +1376,70 @@ do
         assert.same("Bypass", res.headers["X-Cache-Status"])
       end)
     end)
+
+    it("ignore uri case in cache_key", function()
+      local res = assert(client:send {
+        method = "GET",
+        path = "/ignore-case/kong",
+        headers = {
+          host = "route-20.com",
+        },
+      })
+
+      -- here 404 is return by upstream
+      local body1 = assert.res_status(404, res)
+      assert.same("Miss", res.headers["X-Cache-Status"])
+
+      local cache_key1 = res.headers["X-Cache-Key"]
+      assert.matches("^[%w%d]+$", cache_key1)
+      assert.equals(64, #cache_key1)
+
+      local res = client:send {
+        method = "GET",
+        path = "/ignore-case/KONG",
+        headers = {
+          host = "route-20.com",
+        },
+      }
+
+      -- here 404 is return by upstream
+      local body2 = assert.res_status(404, res)
+      assert.same("Hit", res.headers["X-Cache-Status"])
+      local cache_key2 = res.headers["X-Cache-Key"]
+      assert.same(cache_key1, cache_key2)
+
+      assert.same(body1, body2)
+    end)
+
+    it("acknowledge uri case in cache_key", function()
+      local res = assert(client:send {
+        method = "GET",
+        path = "/acknowledge-case/kong",
+        headers = {
+          host = "route-21.com",
+        },
+      })
+
+      -- here 404 is return by upstream
```

Comment:
can we remove these comments too? I feel they are not meaningful here. Should be ready to merge then. Thanks!

Reply:
Yes, I'll remove the comments. 👍